game-core/src/main/java/games/strategy/triplea/settings/ClientSetting.java

```diff
@@ -92,7 +95,11 @@
 
   SHOW_BETA_FEATURES(false),
 
-  SHOW_CONSOLE_ALWAYS(false),
+  SHOW_CONSOLE_ALWAYS(false, saveValue -> {
+    if (saveValue.equals("true")) {
```

Comment:
Nit: Since we're relying on the `String`-to-`boolean` conversions elsewhere, we should probably be consistent and do the same thing here (and guard against an NPE since `ClientSetting#setValue()` allows `null` values).

Reply:
Sorry, forgot the code example:

```java
if (String.valueOf(true).equals(saveValue)) {
```